base/MAPL_VarSpecMod.F90

```diff
@@ -164,6 +173,44 @@ module MAPL_VarSpecMod
   type(MAPL_VarConnType), pointer :: ConnPtr => null()
 end type MAPL_VarConn
 
+type MAPL_VarServiceProviderType
+  private
+  character(len=ESMF_MAXSTR)               :: SERVICE_NAME
+  character(len=ESMF_MAXSTR)               :: BUNDLE_NAME
```

Comment:
Prefer allocatable strings - contingent on whether something else forces these to be fixed length.